server/api/repo.go

```diff
@@ -108,8 +114,12 @@ func PatchRepo(c *gin.Context) {
 		return
 	}
 
-	if (in.IsTrusted != nil || in.Timeout != nil) && !user.Admin {
-		c.String(403, "Insufficient privileges")
+	if in.Timeout != nil && *in.Timeout > maxTimeout && !user.Admin {
+		c.String(http.StatusForbidden, "timeout is higher than max timeout on this instance")
+	}
+	if in.IsTrusted != nil && *in.IsTrusted != repo.IsTrusted && !user.Admin {
```

Comment:
Is this the fix for #534?

Reply:
`&& *in.IsTrusted != repo.IsTrusted` - yes it looks like the new ui send IsTrusted field even if not changed :)

so you could call it a "frontend"-issue but I like to harden the backend so this should just also do/fix it

Reply:
Ahh yip, I changed it to always send the complete from instead of the previous behavior were the form was directly submitted on click or key-input with only the specific part.